proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -10532,6 +10532,15 @@ function bufferRetype( test )
   test.shouldThrowErrorSync( () => _.bufferRetype( new BufferRaw( 5 ) ), I8x, 'extra' );
 }
 
+function bufferRetypeWithOffset( test )
+{
+  test.case = 'F64x from BufferRaw with offset to U16x';
+  var raw = new BufferRaw( 32 );
```

Comment:
Please, use some filled raw buffer. The simplest way:
```js
var raw = new U8x( [ 1, 2, 3, 4, 5, 6, 7, 8 ] ).buffer;
``` 
The size of this buffer is equal to length of passed array. Each byte of this buffer contains number from array.